packages/apps/hr/src/features/expenseSlice.js

```diff
@@ -229,6 +230,19 @@ export const fetchExpenseById = createAsyncThunk(
   },
 );
 
+export const deleteExpenseLine = createAsyncThunk(
```

Comment:
Change name to `deleteExpense` and add it to `asyncFunctions-index.ts` file (export).